### DISCUSSION TITLE
How to change default port?

### DISCUSSION THREAD

**Discussion by Jasenkoo** [nuxt/nuxt#18234] [Questions] [answered]
In v2 we could change it in config:

**nuxt.config.js**

```js
module.exports = {
    server: {
       port: 8080
    }
}
```

I see that in v3 `server` property is allowed, but it is not changing the port and there is nothing in the documentation. I don't know if this is an issue or there is another way to change it.

**Comment by wobsoriano** [comment, marked as answer]
https://github.com/nuxt/framework/discussions/1290#discussioncomment-1510020
